main/src/main/java/cgeo/geocaching/connector/bettercacher/BetterCacherAPI.java

```diff
@@ -292,9 +237,9 @@ public Geocache createGeocache() {
             cache.setCoords(GeopointParser.parse(lat + " " + lng, null));
             // "type": "unknown",
             cache.setType(getFromJson(CACHE_TYPE_MAP, type, CacheType.UNKNOWN));
-            // "difficulty": "4_5",
+            // "difficulty": "4.5",
```

Comment:
Just my few cents:
Is this a left over or for documentation purpose. For the latter it should probably be explained, what it shall tell.

Reply:
Ah OK :)